res/css/style.css

```diff
@@ -195,7 +195,7 @@ body,
   color: black;
 }
 .treeViewRow.dim > .treeViewMainColumn {
-  opacity: 0.7;
+  opacity: 0.6;
```

Comment:
I dimmed it a bit more because otherwise it wouldn't stand out. I checked it still has enough contrast.